docs/whats_new/v1.0.0.md

```diff
@@ -0,0 +1,302 @@
+| CatBoost (2023, 4h8c)        | 95%        | 18.2%               |          0.28 |     6.89 | 3%       |
+| TPOT (2023, 4h8c)            | 91%        | 23.1%               |           0.4 |     8.15 | 1%       |
+| LightGBM (2023, 4h8c)        | 99%        | 23.6%               |           0.4 |     8.95 | 0%       |
+| XGBoost (2023, 4h8c)         | 100%       | 24.1%               |          0.43 |      9.5 | 0%       |
+| RandomForest (2023, 4h8c)    | 97%        | 25.1%               |          0.53 |     9.78 | 1%       |
+
+Not only is AutoGluon more accurate in 1.0,
+it is also more stable thanks to our [new usage of Ray subprocesses](https://github.com/autogluon/autogluon/pull/3614) during low-memory training,
+resulting in **0 task failures** on the AutoML Benchmark.
+
+AutoGluon 1.0 is capable of achieving the fastest inference throughput of any AutoML system while still obtaining state-of-the-art results.
+By specifying the `infer_limit` fit argument, users can trade off between accuracy and inference speed to meet their needs.
+
+As seen in the below plot, AutoGluon 1.0 sets the Pareto Frontier for quality and inference throughput, achieving Pareto Dominance compared to all other AutoML systems.
+AutoGluon 1.0 High achieves superior performance to AutoGluon 0.8.2 Best with 8x faster inference and 8x less disk usage!
+
+![AutoGluon 1.0 AutoML Benchmark Plot](https://raw.githubusercontent.com/Innixma/autogluon-benchmark/master/v1_results/pareto_front_all.png)
+
+We would like to conclude this spotlight by thanking Pieter Gijsbers, Sébastien Poirier, Erin LeDell, Joaquin Vanschoren,
+and the rest of the AutoML Benchmark authors for their key role in 
+providing a shared and extensive benchmark to monitor the progress of the AutoML field.
+Their support has been invaluable to the AutoGluon project's continued growth.
+
+We would also like to thank Frank Hutter, who continues to be a leader within the AutoML field, for organizing
+the AutoML Conference in 2022 and 2023 to bring the community together to share ideas and align on a compelling vision.
+
+We are excited to see what our users can accomplish with AutoGluon 1.0's enhanced performance.
+As always, we will continue to refine and enhance AutoGluon in future releases to push the boundaries of AutoML forward for all.
+
+## Spotlight: Newly Published Papers
+
+### AutoGluon-TimeSeries: AutoML for Probabilistic Time Series Forecasting
+
+We have published a [paper on AutoGluon-TimeSeries](https://openreview.net/forum?id=XHIY3cQ8Tew) at AutoML Conference 2023 ([YouTube Video](https://www.youtube.com/watch?v=niLmfjXeHnE)).
+In the paper, we benchmarked AutoGluon and popular open-source forecasting frameworks (including DeepAR, TFT, AutoARIMA, AutoETS, AutoPyTorch).
+AutoGluon produces SOTA results in point & probabilistic forecasting, and even **achieves 65% win rate against the best-in-hindsight combination of models**.
```

Comment:
Not sure we need a `&` here when we have stuck with `and` everyone else in the notes.

Reply:
added